<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>872</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>442</width>
    <height>837</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notify OSD Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/not.png</normaloff>:/not.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Positioning</string>
      </property>
      <property name="buddy">
       <cstring>positionCombo</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="positionCombo">
      <item>
       <property name="text">
        <string>Fixed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dynamic</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Timeout</string>
      </property>
      <property name="buddy">
       <cstring>timeoutSpin</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="timeoutSpin">
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="suffix">
       <string> sec</string>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Bubble</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Bubble Vertical Gap</string>
         </property>
         <property name="buddy">
          <cstring>bblVerticalGap</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Bubble Horizontal Gap</string>
         </property>
         <property name="buddy">
          <cstring>bblHorizontalGap</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QSpinBox" name="bblVerticalGap">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QSpinBox" name="bblHorizontalGap">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Bubble Corner Radius</string>
         </property>
         <property name="buddy">
          <cstring>bblCornerRadius</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QSpinBox" name="bblCornerRadius">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>37</number>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Bubble Icon Size</string>
         </property>
         <property name="buddy">
          <cstring>bblIconSize</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QSpinBox" name="bblIconSize">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Bubble Gauge Size</string>
         </property>
         <property name="buddy">
          <cstring>bblGaugeSize</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QSpinBox" name="bblGaugeSize">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Bubble Width</string>
         </property>
         <property name="buddy">
          <cstring>bblWidth</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QSpinBox" name="bblWidth">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>3000</number>
         </property>
         <property name="value">
          <number>240</number>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Bubble Background Colour</string>
         </property>
         <property name="buddy">
          <cstring>bblBackColour</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QtColorPicker" name="bblBackColour">
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Bubble Background Opacity</string>
         </property>
         <property name="buddy">
          <cstring>bblOpacity</cstring>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QSpinBox" name="bblOpacity">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Title</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Text Margin Size</string>
         </property>
         <property name="buddy">
          <cstring>marginSize</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="marginSize">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Text Title Size</string>
         </property>
         <property name="buddy">
          <cstring>titleSize</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="titleSize">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Text Title Weight</string>
         </property>
         <property name="buddy">
          <cstring>titleWeight</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="titleWeight">
         <item>
          <property name="text">
           <string>Bold</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Normal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Thin</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Text Title Colour</string>
         </property>
         <property name="buddy">
          <cstring>titleColour</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Text Title Opacity</string>
         </property>
         <property name="buddy">
          <cstring>titleOpacity</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QtColorPicker" name="titleColour"/>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="titleOpacity">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Body</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Text Body Size</string>
         </property>
         <property name="buddy">
          <cstring>bodySize</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Text Body Weight</string>
         </property>
         <property name="buddy">
          <cstring>bodyWeight</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Text Body Colour</string>
         </property>
         <property name="buddy">
          <cstring>bodyColour</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="bodySize">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="bodyWeight">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Bold</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Normal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Thin</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QtColorPicker" name="bodyColour"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Text Body Opacity</string>
         </property>
         <property name="buddy">
          <cstring>bodyOpacity</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Text Shadow Opacity</string>
         </property>
         <property name="buddy">
          <cstring>shadowOpacity</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="bodyOpacity">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="shadowOpacity">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="applyBut">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="cursor">
    <cursorShape>WaitCursor</cursorShape>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QPushButton</extends>
   <header>qtcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
